geometry_msgs/QUALITY_DECLARATION.md

```diff
@@ -82,15 +82,15 @@ There are no currently copyrighted source files in this package.
 
```

Comment:
I noticed that only `4.v` is listed inside testing. Maybe we should list all items?

(the same goes for several other packages)

Reply:
They would all say the same thing. This is an interfaces package and doesn't require them. It seemed simpler to just write that at the top of section 4

Reply:
Fair enough, sounds good